wurst/util/StringUtils.wurst

```diff
@@ -157,6 +126,101 @@ public function string.reduce(string reduced) returns string
 public function string.reduce() returns string
 	return this.reduce(" ")
 
+public interface ForStringSplitCallback
+	function callback(string str)
+
+/** Custom Function which can split strings and runs callback for each string.
+	It can run even with no delimiter present (with single element).
+	Example usage:
+	"H001,H002".forEachIn(",") ->
+		doSomething()  */
+public function string.forEachIn(string delim, ForStringSplitCallback cb)
+	if this.length() > 0
+		int i = 0
+		int pos = this.indexOf(delim, i)
+
+		if pos == -1
+			cb.callback(this.substring(i, this.length()))
+
+		while pos != -1
+			cb.callback(this.substring(i, pos))
+			pos += 1
+			i = pos
+			pos = this.indexOf(delim, pos)
+			if pos == -1
+				cb.callback(this.substring(i, this.length()))
+
+/** Creates list of IDs converted from Int to their String value delimited by comma (using toRawCode).*/
+public function createDelimitedString(vararg int ids) returns string
+	string s = ""
+	var paramCount = 0
+	for id in ids
+		paramCount++
+		s += id.toRawCode()
+		s += ","
+	if s != ""
+		s = s.substring(0, s.length()-1)
+	if paramCount > WC3LIMIT_FUNCTIONS_PARAMETER_COUNT
+		error(WC3LIMIT_FUNCTIONS_PARAMETER_COUNT_ERROR)
+	return s
+
+/** Creates one long String from all Strings delimited by comma.*/
+public function createDelimitedString(vararg string ids) returns string
+	string s = ""
+	var paramCount = 0
+	for id in ids
+		paramCount++
+		if id != ""
+			s += id
+			s += ","
+	if s != ""
+		s = s.substring(0, s.length()-1)
+	if paramCount > WC3LIMIT_FUNCTIONS_PARAMETER_COUNT
+		error(WC3LIMIT_FUNCTIONS_PARAMETER_COUNT_ERROR)
+	return s
+
+/** Checks whether a string is representing a digit or not.*/
+public function string.isDigit() returns bool
```

Comment:
The name is wrong, `digit` refers to a single element of a number, see definition 

> digit
/ˈdɪdʒɪt/
noun
any of the numerals from 0 to 9, especially when forming part of a number.

Also I think it would be easier to check if it's the string "0", and if not, use .toInt/.toReal which return 0 if it's not a valid number.

Reply:
From what I've just tested, that does not have to be the case always. If you mix numbers and letters, it can convert part of those numbers from string to integer/real.
Therefore this function is safer, just needs a few more modifications.